src/component/modal/trackCreateModal/index.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Box, Button, Modal, TextField, Typography,
+} from '@mui/material';
+import { useState } from 'react';
+import { useCreateTrack } from 'query/tracks';
+import * as S from './style';
+
+interface MemberInfoModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const style = {
+  position: 'absolute' as const,
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 600,
+  bgcolor: 'background.paper',
+  boxShadow: 24,
+  p: 4,
+};
+
+export default function TrackCreateModal({ open, onClose }: MemberInfoModalProps): React.ReactElement {
+  const [track, setTrack] = useState({
+    name: '',
+  });
```

Comment:
track 객체의 경우 name이라는 속성 하나만 가지고 있어서 객체말고 다른 방식으로 관리하는 것은 어떨까 생각이 드네요~
`trackName`이라는 `stirng` 타입의 상태로 관리하는 것은 어떤가요?
```suggestion
  const [trackName, setTrackName] = useState<string>('');
```

Reply:
훨씬 간단해지네요!!👍
수정했습니다! f119465a1b9aa907ca3653fcd2be55d60b86dd9a